supertokens_python/framework/django/django_request.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, request: HttpRequest):
         self.request = request
 
     def get_original_url(self) -> str:
-        return self.request.get_raw_uri()
+        return self.request.build_absolute_uri()
```

Comment:
add test for this please where we call an API that calls this function (for example dashboard delete api). Make sure that the API fails for older versions of the django, and passes for the latest version.

Also, is `build_absolute_uri` supported in older versions of django as well? Cause if not, we need to be backwards compatible.

Reply:
Added tests for `DjangoRequest` and `DjangoResponse`. 

`build_absolute_uri` has been Django [since v1.8](https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.build_absolute_uri). This is as far back as the documentation goes.